counter.py

```diff
@@ -0,0 +1,19 @@
+import time
+import keyboard
+
+count = int(input("Enter any integer : "))
```

Comment:
instead of input use argparse libarary for named command line flag. Example:
python counter.py --target 10

Reply:
implemented